frontend/src/features/login/Login.tsx

```diff
@@ -111,6 +111,17 @@ const Login = () => {
             {t('login.forgotYourPassword')}
           </Link>
         </Box>
+        <Box my={2}>
```

Comment:
To make clearer the distinction between login and signup, what about adding a divider and message addressed to users who don't have an account yet?

```suggestion
        <Divider />
        <Box my={2}>
          <Typography variant="subtitle1" gutterBottom>
            No account on Tournesol yet?
          </Typography>
```

(with translation of course)

Reply:
I added your suggestion.
![image](https://user-images.githubusercontent.com/28259/153395787-e033d2e2-693d-49fe-a43d-0a4b099aef49.png)